quad.go

```diff
@@ -0,0 +1,159 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package quad provides numeric integration of single variable functions by quadrature.
+//
+package quad
+
+import (
+	"math"
+	"sync"
+)
+
+// FixedLocationer computes a set of quadrature locations and weights and stores
```

Comment:
Ok (from the programming point of view. From the mathematical point of view no sane rule can request evaluation of the function outside the integration interval.)